.github/workflows/build-test.yml

```diff
@@ -70,11 +70,11 @@ jobs:
           echo "vsix_name=ms-toolsai-jupyter-insiders.vsix" >> $GITHUB_ENV
           echo "test_matrix_os=[\"ubuntu-latest\", \"windows-latest\"]" >> $GITHUB_ENV
 
-      - name: insiders channel
-        # Scheduled builds will publish Insider builds.
-        if: github.event_name == 'schedule' && github.ref == 'refs/heads/main'
-        run: |
-          echo "release_channel=insider" >> $GITHUB_ENV
+    #   - name: insiders channel
```

Comment:
👍 just make sure that we turn it back on with the versions updated when new insiders engine version is out.

Reply:
Will not forget
https://github.com/microsoft/vscode-jupyter/issues/6195